CourseGrab/ViewControllers/Notification/NotificationModalViewController.swift

```diff
@@ -0,0 +1,145 @@
+        subjectCode: "CS",
+        title: "CS 2112: Data Structures and Algorithms, a Very Long Title"
+    )
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        navigationController?.setNavigationBarHidden(true, animated: true)
+        view.backgroundColor = .black
+
+        // Setup appearance
+
+        availableLabel.text = "AVAILABLE NOW"
+        availableLabel.textAlignment = .center
+        availableLabel.adjustsFontSizeToFitWidth = true
+        availableLabel.font = ._24Bold
+        availableLabel.textColor = .courseGrabGreen
+        view.addSubview(availableLabel)
+
+        courseTitleLabel.text = section.title.uppercased()
+        courseTitleLabel.textAlignment = .center
+        courseTitleLabel.adjustsFontSizeToFitWidth = true
+        courseTitleLabel.numberOfLines = 2
+        courseTitleLabel.font = ._24Bold
+        courseTitleLabel.textColor = .white
+        view.addSubview(courseTitleLabel)
+
+        sectionLabel.text = section.section.uppercased()
+        sectionLabel.textAlignment = .center
+        sectionLabel.adjustsFontSizeToFitWidth = true
+        sectionLabel.font = ._20Semibold
+        sectionLabel.textColor = .white
+        view.addSubview(sectionLabel)
+
+        courseIDStackView.spacing = 8
+        courseIDStackView.axis = .vertical
+        view.addSubview(courseIDStackView)
+
+        courseIDTitleLabel.text = "COURSE ID"
+        courseIDTitleLabel.textAlignment = .center
+        courseIDTitleLabel.adjustsFontSizeToFitWidth = true
+        courseIDTitleLabel.font = ._16Bold
+        courseIDTitleLabel.textColor = .courseGrabBorder
+        courseIDStackView.addArrangedSubview(courseIDTitleLabel)
+
+        courseIDLabel.text = "\(section.courseNum)"
+        courseIDLabel.textAlignment = .center
+        courseIDLabel.adjustsFontSizeToFitWidth = true
+        courseIDLabel.font = ._60Bold
+        courseIDLabel.textColor = .white
+        courseIDStackView.addArrangedSubview(courseIDLabel)
+
+        homeButton.setTitle("BACK TO HOME", for: .normal)
+        homeButton.titleLabel?.font = ._18Semibold
+        homeButton.titleLabel?.adjustsFontSizeToFitWidth = true
+        homeButton.setTitleColor(.white, for: .normal)
+        homeButton.on(.touchUpInside, presentHomeViewController)
+        view.addSubview(homeButton)
+
+        studentCenterButton.layer.cornerRadius = 4
+        studentCenterButton.layer.borderWidth = 1
+        studentCenterButton.layer.borderColor = UIColor.white.cgColor
+        studentCenterButton.setTitle("OPEN STUDENT CENTER", for: .normal)
+        studentCenterButton.titleLabel?.font = ._18Semibold
+        studentCenterButton.setTitleColor(.white, for: .normal)
+        studentCenterButton.on(.touchUpInside, openStudentCenter)
+        view.addSubview(studentCenterButton)
+
+        // Setup constraints
+
+        availableLabel.snp.makeConstraints { make in
+            let constaint = UIScreen.main.nativeBounds.height == 1136 ? 40 : 91
```

Comment:
did u mean `constant` or `constraint`?
also is `UIScreen.main.nativeBounds.height == 1136` to check if the user is on `SE`? if so can we extract this into a helper function since you seem to use it in multiple places